CHANGELOG.md

```diff
@@ -151,7 +154,7 @@ Main (unreleased)
   will now default to `data-agent/`, the same default WAL directory as
   Prometheus Agent. (@rfratto)
 
-v0.23.0 (2022-01-13)
+v0.23.0 (2022-02-10)
```

Comment:
This was wrong, v0.22 and v0.23 had the same date. I probably messed this up when refactoring the file.